rewrites.js

```diff
@@ -338,7 +338,7 @@ exports.REWRITES = [
   },
   // Root actions
   {
-    source: '/opencollective/root-actions',
+    source: '/opencollective/root-actions/:section?',
```

Comment:
Not useful for now, but I plan on exposing the various sections on dedicated URLs (`/opencollective/root-actions/clear-cache`)